index.js

```diff
@@ -191,20 +191,25 @@ module.exports.processV3 = (output, origin, locales = []) => {
 
   let audio_track = null;
   let silent_video = false;
-  if (audio_fragments || audio_url) {
-    if (audio_bitrate === 0 || (audio_bitrate && audio_bitrate <= 10)) {
-      // YouTube will return an empty audio track for silent videos.
-      // This track has an extremely low ABR (<10k) and our gstreamer pipeline fails to play it.
-      // If we get one of these, let the box know that this is a silent video.
-      // Typical ABR: mp3 is 96k-320k, spotify is 96k-160k, very bad audio can be as low as 30k)
-      //
-      // If abr exists (is not null or undefined) and is <= 10, then don't return this audio track
-      silent_video = true;
-    } else if (audio_fragments) {
-      const audioManifest = generateManifest(data, true);
-      audio_track = { type: 'manifest', manifest: audioManifest, format_id: audio_format, protocol: audio_protocol };
-    } else {
-      audio_track = { type: 'url', url: audio_url, format_id: audio_format, protocol: audio_protocol };
+
+  if (audioTrack != null) {
+    const { fragments: audio_fragments, url: audio_url, format_id: audio_format, abr: audio_bitrate, protocol: audio_protocol, language } = audioTrack;
+    const audio_language = language || 'unknown';
```

Comment:
Interesting, I guess for movies you will actually get different language tracks?

Reply:
I have only found a single video with alternate language tracks: https://www.youtube.com/watch?v=_lsC0aXyY6g

There's actually a dubbed spanish track. 

This video has been on my testing list for ages, so it 100% did not previously have a spanish language option.